Library/Homebrew/extend/kernel.rb

```diff
@@ -65,7 +65,7 @@ def oh1(title, truncate: :auto)
   sig { params(message: T.any(String, Exception)).void }
   def opoo(message)
     Tty.with($stderr) do |stderr|
-      if ENV["HOMEBREW_GITHUB_ACTIONS"].present?
+      if GitHub::Actions.env_set?
```

Comment:
```suggestion
      if GitHub::Actions.env_set? && ENV["HOMEBREW_TESTS"].blank?
```
Should probably fix the tests.

Reply:
@carlocab Would rather the tests were fixed up properly. This can be done post-merge, have reverted this PR back to 1c39009 for now to unblock the release.